VARIABLES.md

```diff
@@ -44,9 +44,9 @@ Default:  false
 
 ***
 
-### custom_log4j
+### log4j_rolling_enabled
```

Comment:
think on backwards compatibility w this variable, is there really much gained by changing the variable to log4j_rolling_enabled... probably not